cmd/plugins/topology-aware/policy/pools.go

```diff
@@ -588,6 +589,25 @@ func (p *policy) allocatePool(container cache.Container, poolHint string) (Grant
 	return grant, nil
 }
 
+// allowAllocatedCPUs pins container's CPUs according to what has been
+// allocated for it, taking into account if the container should run
+// with hyperthreads hidden.
+func (p *policy) allowAllocatedCPUs(container cache.Container, allocated cpuset.CPUSet, info string) {
```

Comment:
nitpick: I'd probably call this set* since it is essentially a filter+SetCpusetCpus()... maybe call it `setAllocatedCpus` or `setPreferredCpusetCpus`.